esp/smc/SMCLib/TpWrapper.cpp

```diff
@@ -1970,6 +1970,38 @@ void CTpWrapper::getAttPath(const char* Path,StringBuffer& returnStr)
     JBASE64_Decode(Path, returnStr);
 }
 
+#ifdef _CONTAINERIZED
```

Comment:
minor: In general I prefer minimizing the amount of conditionally compiled code - because none of this code will be checked by smoke test (or static analysis).  That is slightly tricky in this case because of the dependency on the interfaces, but could be done by making everything except for the esp/services/CMakeLists.txt change.  @afishbeck what is your preference?

Reply:
Yes, in this particular case we could always build this service plugin, but only load it in containerized configurations.  If we don't mind the minimal extra build overhead.  I like the idea if it means benefitting from automated testing / analysis.